src/ignore-pipe-errors.js

```diff
@@ -1,5 +1,7 @@
 module.exports = function ignorePipeErrors(proc) {
-  proc.stdin.on('error', () => {});
-  proc.stdout.on('error', () => {});
-  proc.stderr.on('error', () => {});
+  if (proc.stdin && proc.stdout && proc.stderr) {
```

Comment:
Shouldn't this be

```js
if (proc.stdout) { proc.stdout.on('error', () => {}); }
if (proc.stderr) { proc.stderr.on('error', () => {}); }
if (proc.stdin) { proc.stdin.on('error', () => {}); }
```

Reply:
Addressed in https://github.com/apiaryio/dredd/pull/1069/commits/f8117e78ac2dcee4b7a103733c9d215614023394.